gradle.properties

```diff
@@ -3,7 +3,7 @@ pluginGroup=com.drinchev.projectlabel
 pluginName=Project Label
 pluginRepositoryUrl=https://github.com/drinchev/project-label
 # SemVer format -> https://semver.org
-pluginVersion=1.1.1
+pluginVersion=1.2.0
```

Comment:
I guess this is the only manual change we need to do before we create the GitHub release

Reply:
Nice, would be cool if even this could somehow be automated 😉